config/vufind/Blender.ini

```diff
@@ -0,0 +1,231 @@
+; Blender configuration. Note that all backends enabled here must have their own
+; configuration set up properly first, i.e. VuFind must be configured to use each
+; individual backend first before Blender can be used.
+
+; Enabled backends and their labels. A label is a translatable string that is
+; displayed in the results list for each result. Use an empty string for no label.
+; Backend order defines the order the results are fetched from each backend.
+[Backends]
+;Solr = "Local Index"
+;Primo = "CDI"
+;EDS = "EDS"
+
+[Blending]
+; Backends used for blending the first results for at most the initial 20 results:
+initialResults[] = Solr
+initialResults[] = Solr
+initialResults[] = Primo
+initialResults[] = EDS
+initialResults[] = Primo
+initialResults[] = EDS
+
+; The number of records to take from each backend when filling a result list in a
+; round-robin fashion:
+blockSize = 7
+
+; Block size based on result count (result count range[colon]block size):
+;adaptiveBlockSizes[] = "0-20:3"
+;adaptiveBlockSizes[] = "21-10000:5"
+
+; ---------- searches.ini settings ----------
+
+[General]
+default_handler = AllFields
+default_sort = "relevance,id asc"
+default_view = list
+default_limit = 20
+;limit_options        = 10,20,40,60,80,100
+case_sensitive_bools = true
+case_sensitive_ranges = true
+
+default_top_recommend[] = SpellingSuggestions
+;default_top_recommend[] = VisualFacets:Visual_Settings
+default_side_recommend[] = SideFacetsDeferred:Results:CheckboxFacets:Blender
+;default_noresults_recommend[] = SwitchTab
+default_noresults_recommend[] = SwitchType
+default_noresults_recommend[] = SwitchQuery:::fuzzy
+default_noresults_recommend[] = SpellingSuggestions
+default_noresults_recommend[] = RemoveFilters
+
+; Set this to true in order to highlight keywords from the search query when they
+; appear in fields displayed in search results.
+highlighting = true
+
+; Set this to restrict the list of fields that will be highlighted (the hl.fl
+; Solr parameter); default = '*' for all fields:
+highlighting_fields = "author,author2,author_additional,author_fuller,container_title,contents,culture,description,dewey-full,edition,era,genre,geographic,institution,isbn,issn,language,lccn,material,measurements,publication_place_txt_mv,publisher,series,series2,title,title_alt,title_full,title_new,title_old,topic"
```

Comment:
This seems like perhaps a local configuration that has crept into the defaults; pretty sure there are some non-standard fields in this list.

Reply:
Fixed.